src/lessons/08_grid_layout/grid-layout.slides.html

```diff
@@ -0,0 +1,758 @@
+                    <li>Group 2 - Property: grid-auto-flow</li>
+                    <li>Group 3 - Propery: grid</li>
+                    <li>Group 4 - Topic: How to declare lengths/sizes (px, rem, %, min-content, max-content, auto, fr), Function: minmax()</li>
+                    <li>Group 5 - Function: repeat()</li>
+                </ol>
+            </section>
+
+            <section>
+                <h1>Device Adaptation</h1>
+            </section>
+
+            <section>
+                <h3>Media Queries</h3>
+                <pre><code class="formattedCode">
+
+              # CSS
+              .grid-container{
+                display: grid;
+                grid-template-columns: 250px 3fr 2fr;
+               }
+               .grid-item-1 {
+                grid-column: 1 / 4;
+               }
+               .grid-item-2 {
+                grid-row: 2 / 4;
+               }
+
+               @media only screen and (max-width: 1000px){
+                 .grid-container{
+                  grid-template-columns: 1fr 1fr;
+                 } 
+                 .grid-item-1 {
+                  grid-column: 1 / 3;
+                  grid-row: 2 / 3;
+                 } 
+                 .grid-item-2 {
+                  grid-row: 1 / 1;
+                 }
+               }
+  
+            # HTML
+            <div class="grid-container">
+              <div class="grid-item-1">Item 1</div>
+              <div class="grid-item-2">Item 2</div>
+              <div class="grid-item-3">Item 3</div>
+              <div class="grid-item-4">Item 4</div>
+              <div class="grid-item-5">Item 5</div>
+            </div></code><br>
+            <div class="grid-container grid-16">
+              <div class="grid-item-1">Item 1</div>
+              <div class="grid-item-2">Item 2</div>
+              <div class="grid-item-3">Item 3</div>
+              <div class="grid-item-4">Item 4</div>
+              <div class="grid-item-5">Item 5</div>
+            </div>
+            </pre>
+            </section>
+
+            <section>
+                <h3>Some Best Practices</h3>
+                <ul>
+                    <li>Use grid with flexbox, not instead of it</li>
+                    <li>Take advantage of negative line numbers</li>
+                    <li>Name your grid lines</li>
+                </ul>
+            </section>
+
+            <section>
+                <h3>Homework</h3>
+                <p>The "ReDI News" team wants to setup an online version of their newspaper. They ask you the help them with this task. As they would like to reach as many readers as possible, the page should be easily readable on a laptop and phone screen.</p>
+            </section>
+
+            <section>
+                <h3>Homework</h3>
+                <p>The team would like you to create a single-page application, the start page, which comprises the following content:
```

Comment:
Not sure that all of them are aware of what a 'single page application' is. We can say maybe a page or website. There is no real navigation so it should be fine.

Reply:
Good point, I will adapt it 👍